connections/connectionwindow.cpp

```diff
@@ -492,8 +501,18 @@ CANConnection* ConnectionWindow::create(CANCon::type pTye, QString pPortName, QS
             //set up the debug console to operate if we've selected it. Doing so here allows debugging right away during set up
             connect(conn_p, SIGNAL(debugOutput(QString)), this, SLOT(getDebugText(QString)));
         }
-        /*TODO add return value and checks */
+        
         conn_p->start();
+
+        connect(this, SIGNAL(updateBusSpeed(int, int)), conn_p, SLOT(updateBusSpeed(int, int)));
```

Comment:
`connect(this, &ConnectionWindow::updateBusSpeed, conn_p, &CANConnection::updateBusSpeed);`

Gives errors during compile time instead of runtime.